src/late_multicellular_stage/editor/MetaballEditorComponentBase.cs

```diff
@@ -708,8 +709,8 @@ protected void GetMouseMetaball(out Vector3 position, out TMetaball? metaball, f
     ///     TODO: this is not implemented currently and just returns the given primary position
```

Comment:
Since the function is now implemented, I think that this comment should be updated